cluster/manifests/storageclass/storageclass.yaml

```diff
@@ -7,9 +7,16 @@ metadata:
   labels:
     kubernetes.io/cluster-service: "true"
     volume-type: {{ if eq .Cluster.ConfigItems.storageclass_standard_gp3 "true" }}gp3{{else}}gp2{{end}}
+    zones-sanitized: "{{ if eq .Cluster.ConfigItems.storageclass_standard_sanitized_zones "true" }}true{{else}}false{{end}}"
```

Comment:
You would need to roll this change out first in a separate PR, otherwise the deletion would not match as no storage class would have the label.

Reply:
Thanks, I realised that too. But instead of a different PR I can just wait with flipping the config item to `"true"`.

Given that initially `storageclass_standard_sanitized_zones: "false"` merging this PR labels the SC with `zones-sanitized: "false"` and doesn't delete it as well. Flipping the config item will then delete the old SC and create the new one with `zones-sanitized: "true"` so it won't be deleted again.